sql/src/test/java/io/crate/planner/node/ddl/ResetSettingsPlanTest.java

```diff
@@ -0,0 +1,152 @@
+ * information regarding copyright ownership.  Crate licenses this file
+ * to you under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.  You may
+ * obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied.  See the License for the specific language governing
+ * permissions and limitations under the License.
+ *
+ * However, if you have executed another commercial license agreement
+ * with Crate these terms will supersede the license and you may use the
+ * software solely pursuant to the terms of the relevant commercial
+ * agreement.
+ */
+
+package io.crate.planner.node.ddl;
+
+import io.crate.analyze.SymbolEvaluator;
+import io.crate.data.Row;
+import io.crate.expression.symbol.Literal;
+import io.crate.expression.symbol.Symbol;
+import io.crate.metadata.Functions;
+import io.crate.metadata.SearchPath;
+import io.crate.metadata.TransactionContext;
+import io.crate.metadata.settings.SessionSettings;
+import io.crate.planner.operators.SubQueryResults;
+import io.crate.test.integration.CrateUnitTest;
+import org.elasticsearch.common.settings.Settings;
+import org.junit.Test;
+
+import java.util.Map;
+import java.util.Set;
+import java.util.function.Function;
+
+import static io.crate.planner.node.ddl.ResetSettingsPlan.buildSettingsFrom;
+import static org.hamcrest.Matchers.is;
+
+public class ResetSettingsPlanTest extends CrateUnitTest {
+
+    @Test
+    public void testResetSimple() throws Exception {
+        Set<Symbol> settings = Set.of(Literal.of("stats.enabled"));
+
+        Settings expected = Settings.builder()
+            .put("stats.enabled", (String) null)
+            .build();
+
+        assertThat(buildSettingsFrom(settings, symbolEvaluator(Row.EMPTY)), is(expected));
+    }
+
+    @Test
+    public void testReset() throws Exception {
+        Set<Symbol> settings = Set.of(Literal.of("stats"));
+
+        Settings expected = Settings.builder()
+            .put("stats.breaker.log.operations.limit", (String) null)
+            .put("stats.breaker.log.operations.overhead", (String) null)
+            .put("stats.breaker.log.jobs.limit", (String) null)
+            .put("stats.breaker.log.jobs.overhead", (String) null)
+            .put("stats.enabled", (String) null)
+            .put("stats.jobs_log_size", (String) null)
+            .put("stats.jobs_log_expiration", (String) null)
+            .put("stats.jobs_log_filter", (String) null)
+            .put("stats.jobs_log_persistent_filter", (String) null)
+            .put("stats.operations_log_size", (String) null)
+            .put("stats.operations_log_expiration", (String) null)
+            .put("stats.service.interval", (String) null)
```

Comment:
wouldn't one or two settings be enough?

Reply:
This was taken from the original unit test, is it worth simplifying ?

Reply:
yes please to reduce useless noise..

Reply:
Actually, these are all the settings `stats` expands to, so i think it is worth to have them in the unit test.